### PR TITLE
Test coverage

### LINKED ISSUE #10: [test coverage] Measure test coverage and add coverage badge in README

### PR DESCRIPTION
**Pull request by gr2m**
- replaces tape with tap (4650ea6)
- adds nyc for coverage

closes #10

- [ ] update https://github.com/eHealthAfrica/kazana/issues/131 when merged

### CODE REVIEW

**Comment by gr2m** [comment]
I’ll simplify the tests, leave it open for now

**Comment by gr2m** [comment]
ready

File: tests/utils/server.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
+var path = require('path')
+
+var _ = require('lodash')
+var startServer = require('kazana-server')
+
+var defautOptions = {
+  name: 'test',
+  config: {
+    backend: {
```
Comment (jo) [inline tests/utils/server.js]: Does this need `kazana` namespace?

Comment (gr2m) [inline tests/utils/server.js]: fixed

**Comment by jo** [comment]
:+1:  LGTM

**Pull request merged**